libsqlite3-sys/build.rs

```diff
@@ -242,7 +242,7 @@ mod build_bundled {
             cfg.flag("-DHAVE_LOCALTIME_R");
         }
         // Target wasm32-wasi can't compile the default VFS
-        if is_compiler("wasm32-wasi") {
+        if env::var("TARGET") == Ok("wasm32-wasi".to_string()) || is_compiler("wasm32-wasi") {
```

Comment:
Suggestion: With the following change, `.to_string()` wouldn't be necessary:

```suggestion
        if env::var("TARGET").as_deref() == Ok("wasm32-wasi") || is_compiler("wasm32-wasi") {
```